@stellar/design-system/webpack.config.js

```diff
@@ -9,11 +9,21 @@ module.exports = {
   output: {
     filename: "[name].js",
     path: path.resolve(__dirname, "build"),
-    libraryTarget: "commonjs-module",
+    libraryTarget: "umd",
   },
   resolve: {
     extensions: [".js", ".json", ".ts", ".tsx"],
   },
+  externals: {
+    react: {
+      commonjs: "react",
+      commonjs2: "react",
+    },
+    "react-dom": {
+      commonjs: "react-dom",
+      commonjs2: "react-dom",
+    },
+  },
```

Comment:
Where I got this idea: 
https://github.com/facebook/react/issues/13991#issuecomment-841509933